src/sentry/snuba/metrics/extraction.py

```diff
@@ -683,14 +688,41 @@ def _get_args_support(fields: Sequence[str], used_in_function: str | None = None
     if len(fields) == 0:
         return SupportedBy.both()
 
-    if used_in_function == "apdex":
-        # apdex can have two variations, either apdex() or apdex(value).
-        return SupportedBy(on_demand_metrics=True, standard_metrics=False)
```

Comment:
I preserve this behaviour within the function called.
<img width="474" alt="image" src="https://github.com/getsentry/sentry/assets/44410/2a90af3b-5da1-4a2c-a2a0-e5454069496d">